preserveData/load_data.rb

```diff
@@ -41,4 +41,38 @@ def load_music_albums
     end
     music_album_arr
   end
+
+  def load_authors
+    authors_arr = []
+    if File.exist?('./storage_data/author.json')
+      authors_data = File.read('./storage_data/author.json')
+      if authors_data != ''
+        JSON.parse(authors_data).map do |author|
+          new_author = Author.new(author['first_name'], author['last_name'], author['id'])
+          authors_arr << new_author
+        end
+      end
+    end
+    authors_arr
+  end
+
+  def load_games
+    games_arr = []
+    if File.exist?('./storage_data/games.json')
+      games_data = File.read('./storage_data/games.json')
+      if games_data != ''
+        JSON.parse(games_data).map do |game|
+          new_game = Game.new(game['multiplayer'], game['last_played_at'], game['id'], game['publish_date'])
+          new_author = Author.new(game['first_name'], game['last_name'])
```

Comment:
I think that actually we are going to change this line with something like `new_author = authors.find { |find_author| find_author.first_name == music_album['first_name'] && find_author.last_name == music_album['last_name'] }` but we can talk about this today in our first meeting. 😎👍